defold-spine/src/comp_spine_model.cpp

```diff
@@ -800,6 +814,43 @@ namespace dmSpine
         dmRender::AddToRender(render_context, &ro);
     }
 
+    static void RenderListFrustumCulling(dmRender::RenderListVisibilityParams const &params)
+    {
+        DM_PROFILE("SpineModel");
+
+        SpineModelWorld* spine_world = (SpineModelWorld*)params.m_UserData;
+
+        const dmIntersection::Frustum frustum = *params.m_Frustum;
+        uint32_t num_entries = params.m_NumEntries;
+        for (uint32_t i = 0; i < num_entries; ++i)
+        {
+            dmRender::RenderListEntry* entry = &params.m_Entries[i];
+            int component_index = entry->m_UserData;
+
+            // get transformations for this entry
+            SpineModelComponent* component_p = spine_world->m_Components.Get(component_index);
+            dmTransform::Transform& trans = component_p->m_Transform;
+
+            const SpineModelBounds& bounds = spine_world->m_BoundingBoxes[component_index];
+
+            // get center of bounding box in local coords
+            float center_x = (bounds.maxX + bounds.minX)/2;
+            float center_y = (bounds.maxY + bounds.minY)/2;
+            dmVMath::Point3 center_local(center_x, center_y, 0);
+            dmVMath::Point3 corner_local(bounds.maxX, bounds.maxY, 0);
+
+            // transform to world coords
+            dmVMath::Vector4 center_world = component_p->m_World * center_local;
+            dmVMath::Vector4 corner_world = component_p->m_World * corner_local;
+
+            float radius =  Vectormath::Aos::length(corner_world - center_world);
```

Comment:
The radius will be the same, no matter what the transform matrix.

Reply:
hmm, why ?

What if i scale the object and make it twice as big ?

The idea is to take two points in local coords (center and corner of box), get their position after transformations, get heir distance and use it as radius.

Reply:
Good point about the scaling though. I agree.

Reply:
ok. Marking this as resolved.